modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/display/context/DLEditFileEntryDisplayContext.java

```diff
@@ -19,12 +19,19 @@
 import com.liferay.portal.kernel.exception.PortalException;
 
 import java.util.Locale;
+import java.util.Map;
 
 /**
  * @author Iván Zaera
  */
 public interface DLEditFileEntryDisplayContext extends DLDisplayContext {
 
+	public default Map<String, Long> getAllMimeTypeSizeLimit()
+		throws PortalException {
+
+		return null;
```

Comment:
Can you return an empty map (`Collections.emptyMap()`) instead of null?

Reply:
I updated!